workflows/runSh.js

```diff
@@ -4,6 +4,7 @@ var self = runSh;
 module.exports = self;
 
 var fs = require('fs-extra');
+var path = require('path');
 
 var getStatusCodeByName = require('../_common/getStatusCodeByName.js');
```

Comment:
using `../` works in requiring modules in windows. tested it on Windows 10 node js 4.8.6